clients/src/main/java/org/apache/kafka/common/compress/ZstdFactory.java

```diff
@@ -26,21 +26,25 @@
 import org.apache.kafka.common.utils.ByteBufferInputStream;
 import org.apache.kafka.common.utils.ByteBufferOutputStream;
 
-import java.io.BufferedInputStream;
 import java.io.BufferedOutputStream;
+import java.io.DataInputStream;
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.nio.ByteBuffer;
 
 public class ZstdFactory {
+    /**
+     * Default compression level
+     */
+    private static final int DEFAULT_COMPRESSION_LEVEL = 3;
```

Comment:
FYI reviewer

This change is a no-op since we are already using the default value of 3 when no value is provided. This change has been made to make it explicit in Kafka code that we are using compression level of 3 with zstd.

Reply:
Since this is unrelated, do we have to include it as part of this PR?

Reply:
Removed this change from this PR.